back/sssscs/src/main/java/com/ib/certificate/CertificateController.java

```diff
@@ -0,0 +1,128 @@
+package com.ib.certificate;
+
+import java.util.List;
+
+import org.springframework.beans.factory.annotation.Autowired;
+import org.springframework.http.HttpStatus;
+import org.springframework.http.ResponseEntity;
+import org.springframework.web.bind.annotation.GetMapping;
+import org.springframework.web.bind.annotation.PathVariable;
+import org.springframework.web.bind.annotation.PostMapping;
+import org.springframework.web.bind.annotation.PutMapping;
+import org.springframework.web.bind.annotation.RequestBody;
+import org.springframework.web.bind.annotation.RequestMapping;
+import org.springframework.web.bind.annotation.RestController;
+
+import com.ib.certificate.CertificateRequest.Status;
+import com.ib.certificate.dto.CertificateRequestCreateDto;
+import com.ib.certificate.dto.CertificateRequestDto;
+import com.ib.certificate.dto.CertificateSummaryItemDto;
+import com.ib.user.IUserService;
+import com.ib.user.User;
+
+@RestController
+@RequestMapping("/api/cert")
+public class CertificateController {
+	@Autowired
+	private ICertificateService certificateService;
+	
+	@Autowired
+	private ICertificateRequestService certificateRequestService;
+	
+	@Autowired
+	private IUserService userService;
+	
+	@PostMapping("/request")
+	public ResponseEntity<String> makeRequest(/*@DTO(CertificateRequestCreateDto.class)*/ @RequestBody CertificateRequestCreateDto certificate) {
+		// TODO: Automatic mapping.
```

Comment:
The reason why dto mapping does not work here is because it's not smart enough to convert a dto's Long attribute representing an id of an entity to the actual entity. We'll need to expand it if we want this to work. Probably have to use Typemaps somehow?